cocos/scene-graph/prefab/utils.ts

```diff
@@ -26,11 +26,10 @@
 import { EDITOR, SUPPORT_JIT } from 'internal:constants';
 import { cclegacy } from '@base/global';
 import { errorID, warn } from '@base/debug';
-import { editorExtrasTag } from '../../core';
+import { editorExtrasTag, ValueType } from '@base/object';
```

Comment:
What's `editorExtrasTag `? Why it belongs to object module?

Reply:
it's a private symbol or string, and this is a js trick to implement `firend` property in js class, for example

```ts
// 'A' module
const editorExtrasTag = symbol();

class A {
    // we use compute property to implement friend feature in js
    [editorExtrasTag] = 1;
}
```

that is any js class can access `editorExtrasTag`, they can access the `[editorExtrasTag]` property in class A

```ts
import { editorExtrasTag } from 'A'
class B {
    a: A = new A();
    accessAFriendProperty () {
        this.a[editorExtrasTag]    // <--
    }
}
```

> Why it belongs to object module?

because `CCObject` implements `EditorExtendableObject`， and `editorExtrasTag` is part of `EditorExtendableObject`